db/migrate/20160310205335_add_moved_to_pieces.rb

```diff
@@ -0,0 +1,5 @@
+class AddMovedToPieces < ActiveRecord::Migration
+  def change
+    add_column :pieces, :moved, :boolean, default: false
```

Comment:
We have method in pawn.rb called first_move? that could take care of this for you without having to add an attribute. So you could just move first_move? to piece.rb and probably just rename it to moved? if you want too.